tools/check-cla

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+author=${1/\/cloud-init/}
+if [ `grep -c ${author} tools/.lp-to-git-user` = 0 ]; then
+    cat <<EOF
+______________________________________________________________
+
+Hello $author,
+
+Thank you for your contribution to cloud-init.
+
+In order to accept this pull request. Please confirm
```

Comment:
This first sentence looks incomplete. Perhaps:

```suggestion
In order to accept this pull request, you need to have signed the Contributor License Agreement (CLA). Please confirm
```

Reply:
Also maybe s/confirm/ensure/ (as confirmation implies that they have signed it in the past, which won't be true for everyone).